test/libsolidity/StandardCompiler.cpp

```diff
@@ -1153,6 +1153,7 @@ BOOST_AUTO_TEST_CASE(linking_yul_same_library_name_different_files)
 
 BOOST_AUTO_TEST_CASE(evm_version)
 {
+	using langutil::EVMVersion;
```

Comment:
If you do that, you can also replace `langutil::EVMVersion::allVersions()` below with `EVMVersion::allVersions()`.

But I'd generally just add `using namespace langutil;` at the top of the file.